src/Sentry.Unity.iOS/SentryNativeCocoa.cs

```diff
@@ -14,15 +15,14 @@ public static class SentryNativeCocoa
     /// Configures the native support.
     /// </summary>
     /// <param name="options">The Sentry Unity options to use.</param>
-    /// <param name="sentryUnityInfo">Infos about the current Unity environment</param>
-    public static void Configure(SentryUnityOptions options, ISentryUnityInfo sentryUnityInfo) =>
-        Configure(options, sentryUnityInfo, ApplicationAdapter.Instance.Platform);
+    public static void Configure(SentryUnityOptions options) =>
```

Comment:
This is a public method, and we only need to pass `options`.
But if the static `SentryPlatformServices.UnityInfo` wasn't set properly before hand, we'll just pass that through below and it'll be hard to trace back why that was.

I think we could validate here (if we expect `SentryPlatformServices.UnityInfo` not to be null here) that it is indeed not null. And deal with it (log and continue, throw?)

Reply:
Nice catch. We're handling this now by having the UnityInfo living on the options and getting validated there.